File: Data Structures & Algorithms/missing-number/submission-1.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        uint32_t s = nums.size();
        for (int i = 0; i < nums.size(); i++) {
            s ^= i;
            s ^= nums[i];
        }
        return s;
    }
};

File: Data Structures & Algorithms/top-k-elements-in-list/submission-1.py
class Solution:
    def topKFrequent(self, nums: List[int], k: int) -> List[int]:
        hashmap = {}
        for x in nums:
            hashmap[x] = hashmap.get(x, 0) + 1

        arr = []
        for num, cnt in hashmap.items():
            arr.append([cnt,num])
        arr.sort()
            
        result = []
        while len(result) < k:
            result.append(arr.pop()[1])

        return result
            